tests/python/sdk/test_auto_annotation.py

```diff
@@ -304,6 +346,88 @@ def detect(context, image: PIL.Image.Image) -> list[models.LabeledShapeRequest]:
             assert self.cat_sublabels_by_id[elements[1].label_id].name == "tail"
             assert elements[1].points == [30, 30]
 
+    def test_detection_attributes(self):
+        spec = cvataa.DetectionFunctionSpec(
+            labels=[
+                cvataa.skeleton_label_spec(
+                    "cat",
+                    123,
+                    [
+                        cvataa.keypoint_spec(
+                            "head",
+                            10,
+                            attributes=[
+                                cvataa.number_attribute_spec(
+                                    "size", 1, number_attribute_values(1, 10, 1)
+                                ),
+                                cvataa.text_attribute_spec("orientation (should be ignored)", 2),
+                            ],
+                        ),
+                        cvataa.keypoint_spec("tail", 30),
+                    ],
+                    attributes=[
+                        cvataa.select_attribute_spec("color", 1, ["gray", "calico"]),
+                        cvataa.text_attribute_spec("name (should be ignored)", 2),
+                    ],
```

Comment:
`*_attribute_spec` - probably, id should be optional. Probably, they are unique within a label.